src/modules/data/blocks/datetime/reducer.js

```diff
@@ -21,8 +21,9 @@ const defaultStartTime = globals.defaultTimes().start ? globals.defaultTimes().s
 const defaultEndTime = globals.defaultTimes().end ? globals.defaultTimes().end : '17:00:00';
 const defaultStartTimeSeconds = time.toSeconds( defaultStartTime, time.TIME_FORMAT_HH_MM_SS );
 const defaultEndTimeSeconds = time.toSeconds( defaultEndTime, time.TIME_FORMAT_HH_MM_SS );
+const queryStartDate = globals.postObjects().tribe_events.tribe_start_date;
 
-export const defaultStartMoment = moment().startOf( 'day' ).seconds( defaultStartTimeSeconds );
+export const defaultStartMoment = queryStartDate ? moment( queryStartDate ).seconds( defaultStartTimeSeconds ) : moment().startOf( 'day' ).seconds( defaultStartTimeSeconds ); // eslint-disable-line max-len
```

Comment:
Use `// eslint-disable-line` as a last resort if you cannot break code into multiple lines (for example, a long string of text that cannot be broken into 2 lines). This, you can break into multiple lines:

```suggestion
export const defaultStartMoment = queryStartDate
	? moment( queryStartDate ).seconds( defaultStartTimeSeconds )
	: moment().startOf( 'day' ).seconds( defaultStartTimeSeconds );
```